### ISSUE TITLE
`get_loglikelihood()` is slow because it calls `get_predicted()` with CIs

### ISSUE THREAD

**Issue by vincentarelbundock** [easystats/insight#585]
This is very slow, even for logit models with only 10K observations, and it slows down a bunch of things downstream in `performance` et al.

We could call `get_predicted(... ci = NULL)` to avoid computing CIs.

**Comment by strengejacke** [comment]
Furthermore, profiled CIs are slow (from `model_parameters()`), that's why we have:

```r
  if (insight::n_obs(model) > 1e4 && identical(ci_method, "profile")) {
    message(insight::format_message("Profiled confidence intervals may take longer time to compute. Use 'ci_method=\"wald\"' for faster computation of CIs."))
  }
```

(https://github.com/easystats/parameters/blob/bd354f9f634e690bd8a51b0ebb39e17cc1a632a7/R/1_model_parameters.R#L623-L625)

**Comment by strengejacke** [comment]
@easystats/maintainers Would you say it makes sense to set `ci = NULL` by default in `get_predicted()`?

**Comment by bwiernik** [comment]
So long as we don't set it to NULL in modelbased, I'm fine with get_predicted() being NULL by default

**Comment by DominiqueMakowski** [comment]
> So long as we don't set it to NULL in modelbased

I agree, insight by default should be fast and "minimal" and then we can leverage and re-purpose its functionalities downstream

**Comment by vincentarelbundock** [comment]
Thanks all!

I didn't know about wald vs profile, so that is very useful info.

One thing to consider is that changing the default value of `ci` will probably break some people's code, and that's something we should probably avoid. We could just be more explicit when we use that function internally (which would be a good thing anyway).

**Comment by strengejacke** [comment]
> One thing to consider is that changing the default value of `ci` will probably break some people's code, and that's something we should probably avoid. We could just be more explicit when we use that function internally (which would be a good thing anyway).

The function is rather new, and I'm not sure it's used that much outside _modelbased_ and _marginaleffects_. So _if_ we decide to change the default to `NULL`, it would be better now than later ;-)

**Comment by vincentarelbundock** [comment]
Good point. Happy to defer to your preferences here.
